src/main/java/com/knighttodo/knighttodo/rest/mapper/TodoRestMapper.java

```diff
@@ -14,12 +14,16 @@
 
     TodoVO toTodoVO(CreateTodoRequestDto requestDto);
 
-//    @Mapping(target = "ready", source = "data", qualifiedByName = "toVerifiableCredential")
+    @Mapping(target = "ready", source = "ready")
     CreateTodoResponseDto toCreateTodoResponseDto(TodoVO todoVO);
```

Comment:
Do you really need @Mapping annotation here?

Reply:
somehow it doesn't map ready properly but I'll check it one more time.